apps/src/tests/TestSplitView/index.tsx

```diff
@@ -24,7 +27,7 @@ const App = () => {
   }
 
   return (
-    <SplitViewBaseApp splitViewBaseConfig={splitViewBaseConfig} />
+    <SplitViewWithNativeStackBase splitViewBaseConfig={splitViewBaseConfig} />
```

Comment:
Why does it change here?

Reply:
Is it rename or you changed the example for some reason?

Reply:
https://github.com/software-mansion/react-native-screens/pull/3282/commits/badbd95435b898898b458373804f782a2934bb66